sandbox-backend/src/main/resources/db/migration/V1_1_0__demo_migration.sql

```diff
@@ -5,4 +5,102 @@ CREATE TABLE IF NOT EXISTS `DEMO_TABLE`
     `SOME_FIELD` varchar(20)
 
 ) ENGINE = InnoDB
-  DEFAULT CHARSET = UTF8;
\ No newline at end of file
+  DEFAULT CHARSET = UTF8;
+
+CREATE TABLE location
+(
+    id         INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    city       VARCHAR(100),
+    modified   DATETIME,
+    modifiedBy VARCHAR(50)
+);
+
+CREATE TABLE employee
+(
+    id         INT                        NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    login      VARCHAR(50)                NOT NULL,
+    password   VARCHAR(255)               NOT NULL,
+    role       enum ('user', 'moderator') NOT NULL,
+    locationId INT,
+    modified   DATETIME,
+    modifiedBy VARCHAR(50),
+    CONSTRAINT FK_employee_location
+        FOREIGN KEY (locationId) REFERENCES location (id),
+    KEY (id)
+);
+
+CREATE TABLE company
+(
+    id         INT         NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name       VARCHAR(50) NOT NULL,
+    modified   DATETIME,
+    modifiedBy VARCHAR(50)
+);
+
+CREATE TABLE company_location
+(
+    id         INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    companyId  INT NOT NULL,
+    locationId INT NOT NULL,
+    FOREIGN KEY (companyId)
+        REFERENCES company (id),
+    FOREIGN KEY (locationId)
+        REFERENCES location (id),
+    CONSTRAINT FK_company_location
+        UNIQUE (companyId, locationId)
+);
+
+CREATE TABLE discount
+(
+    id          INT          NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name        VARCHAR(50)  NOT NULL,
+    description VARCHAR(255) NOT NULL,
+    periodStart DATETIME     NOT NULL,
+    periodEnd   DATETIME     NOT NULL,
+    quantity    INT DEFAULT 0,
+    promocode   VARCHAR(50)  NOT NULL,
+    companyId   INT,
+    modified    DATETIME,
+    modifiedBy  VARCHAR(50),
+    foreign key (companyid) references company (id)
+);
+
+CREATE TABLE tag
+(
+    id      INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    tagName VARCHAR(50)
+);
+
+CREATE TABLE discount_tag
+(
+    id         INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    discountId INT NOT NULL,
+    tagId      INT NOT NULL,
+    FOREIGN KEY (discountId) REFERENCES discount (id),
+    FOREIGN KEY (tagId) REFERENCES tag (id),
+    CONSTRAINT FK_discount_tag
+        UNIQUE (discountId, tagId)
+);
+
+CREATE TABLE employee_discount
+(
+    id         INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    employeeId INT NOT NULL,
+    discountId INT NOT NULL,
+    FOREIGN KEY (employeeId) REFERENCES employee (id),
+    FOREIGN KEY (discountId) REFERENCES discount (id),
+    CONSTRAINT FK_employee_discount
```

Comment:
I think that employee can use discount several times. And all discounts are available to all employees by default